brjs-sdk/sdk/libs/javascript/br-test/src/br/test/GwtTestRunner.js

```diff
@@ -243,6 +243,15 @@ GwtTestRunner.prototype.endTest = function() {
 
 	}
 
+	if (this.m_oFixtureFactory.tearDown) {
+		try {
+			this.m_oFixtureFactory.tearDown();
+		} catch (e) {
+			throw new Errors.CustomError("TestTearDownError", e.message,
```

Comment:
This is how the rest of the errors are handled in GWTTestRunner, I tried to keep it consistent - not sure why it was originally done like this, could be to avoid jstestdriver garbling errors, which its notorious for doing